credsweeper/scanner/scan_type/scan_type.py

```diff
@@ -154,11 +154,16 @@ def _get_candidate(cls, config: Config, rule: Rule, target: AnalysisTarget) -> O
             remove current line. None otherwise
 
         """
+        if target.line.strip() in config.exclude_lines:
+            return None
+
         line_data = cls.get_line_data(config, target.line, target.line_num, target.file_path, rule.patterns[0],
                                       rule.filters)
 
         if line_data is None:
             return None
+        if line_data.value.strip() in config.exclude_values:
```

Comment:
double work in case of user 'denylist'

Reply:
> double work in case of user 'denylist'

Not the case
If we stripped values in list, we still need to strip actual lines, otherwise match cannot accurate